### DISCUSSION TITLE
Blazer com .net8 ma prática

### DISCUSSION THREAD

**Discussion by rodolfoms** [balta-io/forum#478] [Tira Dúvidas] [answered]
Olá Balta
Nesse você citou que faria um crud padrão com cada funcionalidade em um arquivo. Mas que faria um outro mais morderno. Tem esse mais moderno em algum lugar?

Obrigado

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @rodolfoms, como vai? Acho que fizemos ao vivo em um evento, mas não ficou gravado.
Vou colocar no próximo curso de Blazor.
